package.json

```diff
@@ -23,9 +24,8 @@
     "lint:eslint": "eslint . --ext js,cjs,mjs --ignore-path .gitignore",
     "dev": "vite",
     "build:docs": "vite build && cp CNAME site",
-    "eik:login": "eik login",
-    "eik:publish": "eik publish",
-    "eik:alias": "eik pkg-alias",
+    "eik:login": "npx eik login",
```

Comment:
`npx` not needed if the command is part of node_modules

Reply:
Fixed